leasing/models/land_use_agreement.py

```diff
@@ -238,6 +238,9 @@ class LandUseAgreement(TimeStampedSafeDeleteModel):
         verbose_name=_("Plan lawfulness date"), null=True, blank=True
     )
 
+    # In Finnish: Kiinteistöt
```

Comment:
Is "kiinteistöt" the correct term here? It seems "kiinteistöt" (="real estates"/"properties") and "tontit" (="plots") both use the same model which is a bit confusing. I guess this is OK in this context without having to make bigger model changes.

Reply:
Referencing this commit: https://github.com/City-of-Helsinki/mvj/blob/ca29159386dba484f4a572cd97e2e4b85f412410/leasing/models/land_area.py#L316. It depends on the context so here it should be `Kiinteistöt`.